sqlglot/generator.py

```diff
@@ -2083,9 +2083,22 @@ def query_modifiers(self, expression: exp.Expression, *sqls: str) -> str:
             self.sql(expression, "order"),
             *offset_limit_modifiers,
             *self.after_limit_modifiers(expression),
+            *self.sql(expression, "option"),
             sep="",
         )
 
+    def option_sql(self, expression: exp.Option):
+        option_type = self.sql(expression, "option_type")
+        option_value = self.sql(expression, "option_value")
+        if option_value:
+            optional_equal_sign = "= " if expression.args.get("requires_equals") else ""
```

Comment:
So here you'd only add the `=` if the option type exists in the aforementioned set of keywords.

Reply:
Done